### PR TITLE
Change VerifyGitHubVulnerabilities into a job

### PR DESCRIPTION
**Pull request by joelverhagen**
Progress on https://github.com/NuGet/Engineering/issues/4512

The job will run every 6 hours and report a metric which we will monitor on.

### CODE REVIEW

File: src/VerifyGitHubVulnerabilities/Scripts/Functions.ps1 [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Function Uninstall-NuGetService() {
```
Comment (joelverhagen) [inline src/VerifyGitHubVulnerabilities/Scripts/Functions.ps1:1]: Copied from other jobs.

File: src/VerifyGitHubVulnerabilities/Scripts/PostDeploy.ps1 [hunk 2]
```diff
@@ -0,0 +1,20 @@
+. .\Functions.ps1
```
Comment (joelverhagen) [inline src/VerifyGitHubVulnerabilities/Scripts/PostDeploy.ps1:1]: Copied from other jobs.

File: src/VerifyGitHubVulnerabilities/Scripts/PreDeploy.ps1 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+. .\Functions.ps1
```
Comment (joelverhagen) [inline src/VerifyGitHubVulnerabilities/Scripts/PreDeploy.ps1:1]: Copied from other jobs.

File: src/VerifyGitHubVulnerabilities/VerifyGitHubVulnerabilities.nuspec [hunk 4]
```diff
@@ -11,5 +11,10 @@
   </metadata>
   <files>
     <file src="bin\$configuration$\**\*.*" target="bin"/>
+
+	<file src="Scripts\Functions.ps1" />
```
Comment (drewgillies) [inline src/VerifyGitHubVulnerabilities/VerifyGitHubVulnerabilities.nuspec:15]: nit: tabs

**Review by drewgillies — DISMISSED** [review]
Awesome. :)

**Review by drewgillies — APPROVED** [review]

**Pull request merged**